cartography/data/jobs/cleanup/oci_import_compartments_cleanup.json

```diff
@@ -0,0 +1,9 @@
+{
+  "statements": [{
+    "query": "MATCH (:OCITenancy{ocid: {OCI_TENANCY_ID}})-[:RESOURCE]->(n:OCICompartment) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+    "iterative": true,
+    "iterationsize": 100
+  }],
+  "name": "cleanup OCICompartment",
```

Comment:
[question] are we missing  OCICompartment mapping cleanup?

See OCICompartment creation with 
WHERE (aa:OCITenancy OR aa:OCICompartment)

Reply:
+1. Please implement a cleanup job for the relationship between `:OCITenacy` and `OCICompartment` nodes. Something like this:

`match (:OCITenancy{ocid: {OCI_TENANCY_ID}})-[r:RESOURCE]->(:OCICompartment) delete r`